### ISSUE TITLE
feature: return test status to GitHub

### ISSUE THREAD

**Issue by gucio321** [NativeCI/Issues#6]
Hi there,
most CIs returns build status to GitHub so that a user/PR autor can see if there is all OK with his code. 
![obraz](https://user-images.githubusercontent.com/73652197/125642453-df4bf5a4-3349-4fb4-8acf-3b913c9058c1.png)

**Comment by paladium** [comment]
Good idea, that is something I was planning to work on as a part of pull request API integration.
